pkg/builder/builder.go

```diff
@@ -735,6 +781,37 @@ func makeNMAContainer(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.Container
 	}
 }
 
+// makeScrutinizeInitContainer builds the spec of the init container that collects
+// scrutinize command
+func makeScrutinizeInitContainer(vscr *v1beta1.VerticaScrutinize) corev1.Container {
+	return corev1.Container{
+		Image: "busybox:latest",
+		Name:  names.ScrutinizeInitContainer,
+		Command: []string{
+			"sh",
+			"-c",
+			"sleep 5",
+		},
+		VolumeMounts: []corev1.VolumeMount{buildScrutinizeVolumeMount(vscr)},
+		Resources:    vscr.Spec.Resources,
+	}
+}
+
+// makeScrutinizeMainContainer builds the spec of the container that will
+// be running after all init containers are completed
+func makeScrutinizeMainContainer(vscr *v1beta1.VerticaScrutinize) corev1.Container {
+	return corev1.Container{
+		Image: "busybox:latest",
+		Name:  names.ScrutinizeMainContainer,
+		Command: []string{
+			"sh",
+			"-c",
+			"sleep infinity",
+		},
+		Resources: vscr.Spec.Resources,
```

Comment:
I don't think we want to set resources for this pod. Or at least we don't want to use the resources set in the spec. This container is meant to spin allowing time to collect the tarball. We can take this as a follow-on task, but I think we should have some resources set, only if the resources are set for the init container.

Reply:
Maybe we can use the same logic as nma resources. For now I will leave it unset.